### DISCUSSION TITLE
Vengeance DH

### DISCUSSION THREAD

**Discussion by MXF2468** [Hekili/hekili#2178] [Q&A]
Hi all.  Apologies in advance if this is a dumb question.  I'm playing vengeance DH, and as far as I can tell, the addon is never recommending stacking frailty before using CDs, even if I have full fury.  This seems like a pretty big dps loss to me, and when I try to "override" it manually (i.e., dumping fury into stacking frailty before using the cds), my dps seems to go up quite a bit.  This seems to occur both in ST and AOE.

I'm not sure if I have something set up wrong, I'm pretty much just using it with the default settings.

If anyone has any suggestions, or has seen something similar, I would appreciate your advice.  Thanks very much!

**Comment by Hekili** [comment]
This would be a topic for the ticket system, where you can include snapshots from in-game along with sims of your character for comparison.  Because of how Frailty "stacks" (overlapping hidden procs), attempting to forecast to such an extent that you delay your CDs can end up being more of a problem.

I'm considering setting a class option for Frailty stacks but it's not at the top of my to-do list.

**Reply by MXF2468** [reply]
Thank you so much for the prompt reply.  I'm not really sure how to do the things you suggest (sorry - not so sophisticated with addons and changing sims and stuff).  I just noticed my dps improves markedly if, when the addon suggests blowing all my cds, I stack frailty first and THEN start following the addon again.  It sounds like it's a challenge to put frailty into the addon, no worries.  I'll just put a subroutine between my ears to handle that.  Really appreciate the reply!  Thank you.
